stacks_queues/stack.py

```diff
@@ -21,18 +21,25 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        v = self.store.get_first()
+        self.store.remove_first()
+        return v
```

Comment:
If you read the docstrings of the LinkedList class, you'll see that `remove_first` returns the value of the node it removes.

```suggestion
        return self.store.remove_first()
```